tests/stdlib/tos.nim

```diff
@@ -614,6 +614,10 @@ block osenv:
     doAssert existsEnv(dummyEnvVar) == false
     delEnv(dummyEnvVar)         # deleting an already deleted env var
     doAssert existsEnv(dummyEnvVar) == false
+  block putEnv:
```

Comment:
```suggestion
  block: # putEnv
```

refs https://nim-lang.github.io/Nim/contributing.html (even though this file uses `block putEnv`, we should prefer the recommended style instead of the style used in the same file); rationale is explained in other placse